### DISCUSSION TITLE
Export as QR Code

### DISCUSSION THREAD

**Discussion by DaFlowah** [ente-io/ente#1019] [Enhancements]
**Is your feature request related to a problem? Please describe.**
Some 2FA apps (like Google Authenticator) support import as QR code.
Currently Ente only supports export as text file.

**Describe the solution you'd like**
Allow export of all codes as a single QR Code.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by DaFlowah** [comment]
Somehow similar to https://github.com/ente-io/auth/issues/460

**Comment by ua741** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/500
